doc/man3/X509_verify.pod

```diff
@@ -70,10 +70,11 @@ The X509_verify(), X509_REQ_verify(), and X509_CRL_verify()
 functions are available in all versions of OpenSSL.
 
 X509_REQ_verify_ex(), and X509_self_signed() were added in OpenSSL 3.0.
+X509_verify_rpk() was added in OpenSSL 3.2.
```

Comment:
this line should be added in the next pod file, no ?

Reply:
perhaps, but the most recent (3rd) commit in my review fork suggests removing this function, I hope that will happen. The functionality will then be subsumed by `X509_verify_cert()` and/or equivalently `X509_STORE_CTX_verify()`

Reply:
I still haven't gotten there yet (to look at it). There are things (believe it or not!) that are taking priority for me over this PR...

Reply:
No worries of course. Looking forward to resuming the review once you do find some time.

Reply:
I pulled in your most recent change.